plugins/Satellites/src/Satellites.cpp

```diff
@@ -288,7 +288,7 @@ bool Satellites::backupCatalog(bool deleteOriginal)
 		return false;
 	}
 
-	QString backupPath = catalogPath + ".old";
+	QString backupPath = catalogPath + ".old";  // FIXME replace with datetime stamp as generic routine to backup files
```

Comment:
If you add a date to the old filename, also add a method to at least warn the user to manually remove the 50 old files that may appear soon and clutter your user dir. And add documentation about how to use old files.

Reply:
Out of scope, this should be tackled in a separate issue, also because there are more places where this logic can be applied (eg log files); also, the presence of many log files can indeed become a burden for the unaware user. So this requires some more reflexion. 

(Also related to #1283 )